src/js/control-bar/seek-to-live.js

```diff
@@ -61,9 +61,13 @@ class SeekToLive extends Button {
    */
   updateLiveEdgeStatus(e) {
     if (this.player_.liveTracker.behindLiveEdge()) {
+      this.setAttribute('aria-disabled', false);
       this.removeClass('vjs-at-live-edge');
+      this.controlText('Seek To LIVE');
```

Comment:
I wonder if this should be more verbose like `not at live edge, click to seek to live`.

Reply:
I am going to change it to that, as being verbose is probably better.

Reply:
Is there a display somewhere in the control bar which shows how far behind live you are? I'm wondering if that ought to also be indicated here, e.g. "Seek to Live, currently 1:23 behind live". I think you should stay away from the term "edge" - that's a technical term, but not one a typical user would recognize.

Reply:
the current time shows up as how far behind live you are, I think. We can probably use that to update this.

Reply:
Also, try to avoid "click to..." - that's implied with a button, but some users may tap, some click, some use the keyboard. And try to front-load the action, then add more information afterwards - "Seek to Live, currently 1:23 behind live" instead of "not currently live, seek to live".

Reply:
Ah, makes sense for the action to be front loaded, I was thinking that maybe the info is more important but it isn't